test/test_jit.py

```diff
@@ -62,6 +62,10 @@
 WINDOWS = sys.platform == 'win32'
 
 
+def pyfn(a, b):
+    return a * b
```

Comment:
Can this be made local to a test? That might really end up being clearer (or you can make it a static method of the test class).

Reply:
Something strange happened the last time I made it local to a test, but I'll try again and file an issue if it errors out.

Reply:
https://github.com/pytorch/pytorch/issues/12393 The scoping behavior on torch.jit.script is a little weird so I couldn't make it local to a test.

Making it a static method of the test class doesn't work either (the compiler complains about "not being able to access members of TestAutodiffSubgraphSlicing")